tokens/tokens-eth.json

```diff
@@ -287,7 +287,39 @@
   "twitter"   : "https://twitter.com/AdEx_Network",
   "youtube"   : ""
 }
-},{
+}, {
+    "symbol": "AE",
+    "address": "0x5ca9a71b1d01849c0a95490cc00559717fcf0d1d",
+    "decimals": 18,
+    "name": "",
```

Comment:
an empty name is quite bad

Reply:
Yes I agree, @stevemieskoski when you merge from current mew list to this make sure you add as much information you can find about that token. Thanks @ligi

Reply:
Ideally let the check-script run - just do './gradlew run' - it will complain if stuff like this is missing. Would also be happy to perhaps get our stuff closer together. I made the data much cleaner in github.com/ethereum-lists. Now ERC-55 everywhere - address trimming (some here have leading or trailing spaces)

Reply:
👍